app/src/main/res/layout/view_item_divider.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2020 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<View style="@style/ListDivider" />
```

Comment:
what's the benefit of this over simply using the `View` with the style when needed?

Reply:
Not sure if I follow the comment 100%, but I guess you mean what's the benefit of this approach vs having a favorite/bookmark item that can show a divider at the bottom part of the view(?).

If that's the question, the benefit is to simplify the logic inside those adapters and avoid any logic to detect the last item on the list in order to show that divider. In this case, the divider is always visible between favorites and bookmarks, and for that use case (because it's static and always visible), it's easier to just declare it in this way.

Reply:
Gotcha, this layout belongs to an itemViewType from the RV. Thanks for the clarification!